modules/star/1.3/star.smk

```diff
@@ -48,38 +52,56 @@ rule _star_input_fastq:
         fastq_2 = CFG["inputs"]["sample_fastq_2"],
     output:
         fastq_1 = CFG["dirs"]["inputs"] + "fastq/{seq_type}--{genome_build}/{sample_id}.R1.fastq.gz",
-        fastq_2 = CFG["dirs"]["inputs"] + "fastq/{seq_type}--{genome_build}/{sample_id}.R2.fastq.gz",
+        fastq_2 = CFG["dirs"]["inputs"] + "fastq/{seq_type}--{genome_build}/{sample_id}.R2.fastq.gz"
     run:
         op.relative_symlink(input.fastq_1, output.fastq_1)
         op.relative_symlink(input.fastq_2, output.fastq_2)
 
+# Function to retrieve read length from sample table
+def get_overhang(wildcards,build = False):
+    tbl = config["lcr-modules"]["star"]["samples"]
+    # read_length = tbl[(tbl.sample_id == wildcards.sample_id) & (tbl.seq_type == wildcards.seq_type)]["read_length"].values()[0]
```

Comment:
this line is probably temporary?

Reply:
Yes, thanks I'll delete that line when I update this to the new version.